src/dom/remove-class.js

```diff
@@ -41,7 +41,7 @@ module.exports = function removeClass( elm, class_name, callback ) {
   }
 
   // remove class via className
-  if ( !removed ) {
+  if ( !removed && elm && elm.className && typeof elm.className.replace === 'function' ) {
```

Comment:
thanks for adding this check. few thoughts:
* the `elm` check is already above so it shouldn’t be necessary here as well.
  * or did you find a case where that check failed to catch an undefined elm?
* did ie run into an issue with not having elm.className available on an svg element?
  * in that case, i’d rather just add a test for elm.className: if it exists, the .replace method should be available as elm.className is supposed to always return a string as far as i know.